portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -20,13 +20,12 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
+/**Servlet that return a greeting for web visitors*/
+@WebServlet("/greeting")
```

Comment:
Should this be hooked up to the "/comments" servlet? The TODO says to modify the file so that it can handle comments data.

Reply:
Yes, I created another one because I was testing how to handle multiple servlets, this file will be deleted.